packages/putty.vm/putty.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>putty.vm</id>
```

Comment:
Install fails, I think we should prefer the existing package: https://community.chocolatey.org/packages/putty